src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,90 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private int size;
+    private Object[] table;
```

Comment:
```suggestion
    private Node<K, V>[] table;
```
use array of your nodes

Reply:
done